src/encoded/static/components/static-pages/HomePage.js

```diff
@@ -125,116 +159,88 @@ class BigBrowseButton extends React.Component {
 }
 
 
-class LinksColumn extends React.PureComponent {
-
-    jointAnalysisPageLink(colSize){
-        var className = "link-block";
-        if (colSize){
-            className += (' col-sm-' + colSize);
-        }
-        return (
-            <div className={className}>
-                <a href="/joint-analysis">
+const GettingStartedLinksRow = React.memo(function GettingStartedLinksRow(props){
+    const { linkBoxVerticalPaddingOffset, session } = props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['linkBoxVerticalPaddingOffset' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4527127)